code/go/0chain.net/sharder/worker.go

```diff
@@ -436,7 +436,8 @@ func (sc *Chain) moveBlockToCloud(ctx context.Context, round int64, hash string,
 }
 
 func (sc *Chain) SharderHealthCheck(ctx context.Context) {
-	const HEALTH_CHECK_TIMER = 60 * 5 // 5 Minute
+	// TODO: Move to a config file
+	const HEALTH_CHECK_TIMER = 1 * time.Minute
```

Comment:
is it changed to 1 miniute?

Reply:
All providers are unified to use a single health check period https://github.com/0chain/0chain/blob/ce8db27011d6f52536a49c692b94fbd030639c7f/code/go/0chain.net/smartcontract/dbs/event/provider.go#L78

Reply:
Which is set to 1 minute
https://github.com/0chain/0chain/blob/3010dc381efd028a6619e8ede41975d2497cbd97/code/go/0chain.net/smartcontract/dbs/event/provider.go#L18

Reply:
why is it changed to 1 minute? it was 5 minutes before, no?

Reply:
That was for testing, reverted.